urbanairship-adapter/src/main/java/com/gimbal/urbanairship/GimbalAdapter.java

```diff
@@ -14,34 +16,37 @@
 
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.annotation.RestrictTo;
+import androidx.annotation.RequiresPermission;
 import androidx.core.content.ContextCompat;
 
+import com.gimbal.android.Attributes;
 import com.gimbal.android.DeviceAttributesManager;
 import com.gimbal.android.Gimbal;
 import com.gimbal.android.PlaceEventListener;
 import com.gimbal.android.PlaceManager;
 import com.gimbal.android.Visit;
 import com.urbanairship.UAirship;
+import com.urbanairship.analytics.CustomEvent;
 import com.urbanairship.analytics.location.RegionEvent;
+import com.urbanairship.json.JsonValue;
 import com.urbanairship.util.DateUtils;
 import com.urbanairship.util.HelperActivity;
 
-import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.CopyOnWriteArrayList;
 
-import static android.Manifest.permission.ACCESS_FINE_LOCATION;
-
 /**
  * GimbalAdapter interfaces Gimbal SDK functionality with Urban Airship services.
  */
 public class GimbalAdapter {
     private static final String PREFERENCE_NAME = "com.urbanairship.gimbal.preferences";
-    private static final String STARTED_PREFERENCE = "com.urbanairship.gimbal.is_started";
-    private static final String API_KEY_PREFERENCE = "com.urbanairship.gimbal.gimbal_api_key";
+    private static final String API_KEY_PREFERENCE = "com.urbanairship.gimbal.api_key";
```

Comment:
I was wondering if there'd be any ramification to changing this key -- looks like airship recommends folks call `restore` on app startup, which uses this preference. Worst-case scenario, if a customer updates their app to a version that uses our adapter, and a user doesn't do a fresh install,`restore` won't work because it'll be looking for the API key using a different preference string. Then again, I have no idea what `restore` best practices are, I'd assume folks would be calling both `restore` _and_ `startAdapter`, which sets this preference to begin with, so hopefully this won't be a big issue.